desktop/src/app.rs

```diff
@@ -55,6 +55,10 @@ impl WinitApp {
 	}
 
 	fn send_messages_to_editor(&mut self, mut responses: Vec<FrontendMessage>) {
+		if responses.is_empty() {
```

Comment:
This should stay where it was previously since the extract if loops could remove messages and the only expensive part about this is sending the cef message